Bottom: 01b56d3f125c467cd8798de62734d65f1e912b0f
Top:    d32e7274c5f57e1d4d9de5a53eec5a0a1d01eefe
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-24 17:05:20 +0000

Revert "Fix crash upon pressing "f" hotkey (find connected) after a board revert."

This reverts commit 7308f512307158944482227d58e66373fd023d62.


---

diff --git a/src/mymem.c b/src/mymem.c
index 3de31f6..faa3adc 100644
--- a/src/mymem.c
+++ b/src/mymem.c
@@ -677,7 +677,6 @@ FreePCBMemory (PCBType *pcb)
   for (i = 0; i <= MAX_FONTPOSITION; i++)
     free (pcb->Font.Symbol[i].Line);
   FreeLibraryMemory (&pcb->NetlistLib);
-  NetlistChanged (0);
   FreeAttributeListMemory (&pcb->Attributes);
   /* clear struct */
   memset (pcb, 0, sizeof (PCBType));
